flixel/tile/FlxTilemap.hx

```diff
@@ -128,9 +128,18 @@ class FlxTilemap extends FlxBaseTilemap<FlxTile>
 	private var _scaledTileHeight:Float = 0;
 	
 	#if FLX_DEBUG
-	private var _debugTileNotSolid:BitmapData;
-	private var _debugTilePartial:BitmapData;
-	private var _debugTileSolid:BitmapData;
+	private var updateDebugTileBoundingBoxesInit = false;
+	
+	override public function setDebugColorScheme(debugColorScheme:DebugColorScheme)
+	{
+		this.debugColorScheme = debugColorScheme;
+		if (updateDebugTileBoundingBoxesInit)
+			updateDebugTileBoundingBoxes();
+	}
+	
+	private var _debugTileNotSolid:BitmapData = null;
```

Comment:
and don't forget to dispose these bitmaps in `destroy()` method. Change lines
```
_debugTileNotSolid = null;
_debugTilePartial = null;
_debugTileSolid = null;
```
to 
```
_debugTileNotSolid = FlxDestroyUtil.dispose(_debugTileNotSolid);
_debugTilePartial = FlxDestroyUtil.dispose(_debugTilePartial);
_debugTileSolid = FlxDestroyUtil.dispose(_debugTileSolid);
```

(this don't have to be the part of this PR, but it will make code a tiny-tiny bit better :)